XamarinCommunityToolkit/Views/GravatarImageSource/UWP/GravatarImageSourceHandler.uwp.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.UI.Xaml.Media.Imaging;
+using Xamarin.Forms.Platform.UWP;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class GravatarImageSourceHandler : IImageSourceHandler
+	{
+		public async Task<Windows.UI.Xaml.Media.ImageSource> LoadImageAsync(Xamarin.Forms.ImageSource imagesource, CancellationToken cancellationToken = default)
```

Comment:
Can you simplify the name removing `Xamarin` prefix? If it's possible can you use an alias instead of the full namespace, to follow our code style?